src/main/java/org/tron/core/capsule/ReceiptCapsule.java

```diff
@@ -134,7 +146,7 @@ private void payEnergyBill(
       account.setBalance(balance - energyFee);
 
       manager.adjustBalance(manager.getAccountStore().getBlackhole().getAddress().toByteArray(),
-          energyFee);//send to blackhole
+          energyFee);//send to blackHole
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this trailing comment on the previous empty line. ([squid:TrailingCommentCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ATrailingCommentCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbSdHjbSgN4uCclEWqX&open=AWbSdHjbSgN4uCclEWqX)

[ISSUE_KEY]: AWbSdHjbSgN4uCclEWqX